scripts/nm-quick.sh

```diff
@@ -175,6 +175,39 @@ install_yq() {
 	fi	
 }
 
+
+set_install_dir() {
+
+	if [ -z $INSTALL_DIR]; then
+	    INSTALL_DIR=/root
+	fi
+
+	if [ -z $AUTO_BUILD ] && [ "$INSTALL_TYPE" != "ee" ]; then
+	    select directory_option in "Default install directory (/root)" "Custom install directory"; do
+		    case $REPLY in
+			1)
+			echo "using default install directory "$INSTALL_DIR" "
+			break
+			;;
+			2)
+			while true
+			do
+			    read -p "Enter your diserd install directory: " GET_INSTALL_DIR
```

Comment:
Desired?

Reply:
@mattkasun are you just asking if this is something we would like to do? I think yes this would be a nice addition.

Reply:
i think it's the typo

Reply:
ahh yes typo (diserd) @STeffaine will wait on fix